elastic/beats
# add .python-verson file

## What does this PR do?

Add .python-version file with the same version we're using on CI.

## Why is it important?

It documents which Python version should be used an allows for the use of pyenv to manage the Python version automatically.
Currently our system tests are not compatible with Python >= 3.10.0

## Checklist


~~## Author's Checklist~~
## How to test this PR locally

Install Pyenv and run
```sh
pyenv version
```

The output should state that the Python version is set by the `.python-version` file on the root of this repository.

~~## Related issues~~

## Use cases

Any engineer trying to run system tests on a system with Python >= 3.10

~~## Screenshots~~
~~## Logs~~
